src/libraries/System.Text.Json/src/System.Text.Json.csproj

```
@@ -18,6 +18,10 @@ The System.Text.Json library is built-in as part of the shared framework in .NET
     <NoWarn Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETCoreApp'">$(NoWarn);nullable</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ILLinkSubstitutionsXmls Include="ILLink\ILLink.Substitutions.xml" />
```

(nit) - to follow the same pattern other projects use.

```suggestion
    <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)\ILLink.Substitutions.xml" />
```

For example:
https://github.com/dotnet/runtime/blob/cd3c6596c1af5a156c75230f096023266202ef94/src/libraries/System.Net.Http/src/System.Net.Http.csproj#L23-L28